released/discovery_center/mission_4327/setup_subaccount_cf/README.md

```diff
@@ -0,0 +1,70 @@
+
+1. Configure the BTP entitlements required to complete the Discovery Center mission
+2. Enables BTP runtime CF
+3. Create the neccessary Subscription to Software as Service(SaaS) Apps (SAP Business Application Studio(BAS), SAP Build Work Zone, standard edition etc)
+4. Assgn users the neccessary roles required to access the SaaS Apps e.g BAS
+5. Add additional users to the subaccount.
+
+## Prerequisites
+
+Before using this script, make sure you have Terraform downloaded and installed.
+
+- **Install Terraform**: If you haven't already, download and install Terraform from [here](https://www.terraform.io/downloads.html).
+
+## BTP Sub Account Setup
+
+- [BTP Subaccount with CF runtime environment](#btp-subaccount-with-cf-runtime-environment)
+
+ ### Entitlements
+
+| Service     |      Plan      |  Quota required |
+| ------------- | :-----------: | ----: |
+| Cloud Foundry Runtime     | MEMORY | 1 |
+| SAP Build Work Zone, standard edition    |  Standard    |   1 |
+| SAP HANA Cloud |   hana    |    1 |
+| SAP HANA Cloud |   tools   |    1 |
+| SAP HANA Schemas & HDI Containers |   hdi-shared   |    1 |
+
+## Deploying the resources
+
+To deploy the resources you must:
+1. Clone repository `git clone https://github.com/SAP-samples/btp-terraform-samples.git`
+2. Navigate to `released/discovery_center/mission_4327/setup_subaccount_cf`
+3. You will be seeing these files named `main.tf`,`provider.tf`,`samples.tfvars`,`variables.tf`.
+4. Change the variables in the samples.tfvars file to meet your requirements(By default free-tier plans are used, if you want to use it for production update in the samples.tfvars accordingly)
+Follow these steps to use the script:
+5. **Install Terraform Plugins**: Open a terminal and navigate to the directory containing your Terraform configuration files. Run the following command to initialize and upgrade Terraform plugins:
+
+    ```shell
+    terraform init -upgrade
+    ```
+
+6. **Review Changes**: Generate an execution plan to review the changes that will be made to your SAP BTP account. Run:
+
+    ```shell
+    terraform plan -var-file="sample.tfvars"
+    ```
+
+7. **Apply Configuration**: Apply the Terraform configuration to create the SAP BTP subaccount and entitlements. Run:
+
+    ```shell
+    terraform apply -var-file="sample.tfvars"
+    ```
+
+    Confirm the changes by typing "yes."
+
+8. **Cleanup**: After your session or project is complete, you can delete the SAP BTP subaccount and associated resources to avoid charges:
+
+    ```shell
+    terraform destroy
+    ```
+
+    Confirm the resource destruction by typing "yes."
+
+9. **Optional**: You can remove the Terraform state file (`terraform.tfstate`) manually if needed.
+
+Please exercise caution when using this script, especially in production environments, and ensure you understand the resources that will be created or modified.
```

Comment:
The samples should **never** be used in production. We do not give any warranty on them.